recipes-extended/xen/files/argo-suppress-debug-messages.patch

```diff
@@ -0,0 +1,58 @@
+################################################################################
+SHORT DESCRIPTION: 
+################################################################################
+Suppress some frequently occurring argo debug messages by default. Only
+print if argo DEBUG is enabled.
+
+################################################################################
+LONG DESCRIPTION: 
+################################################################################
+
+################################################################################
+CHANGELOG 
+################################################################################
+
+################################################################################
+REMOVAL 
+################################################################################
+
+################################################################################
+UPSTREAM PLAN
+################################################################################
```

Comment:
An upstream plan statement is also needed -- please see my comment inline in the code below.